pkgs/by-name/lu/luau-lsp/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "luau-lsp";
+  version = "1.42.0";
+
+  src = fetchFromGitHub {
+    owner = "JohnnyMorganz";
+    repo = "luau-lsp";
+    tag = finalAttrs.version;
+    hash = "sha256-MX5jXkSxfN7IqY3kvlAKiimFecsuvCnCQ0QegISvAxE=";
+    fetchSubmodules = true;
+  };
+
+  # https://github.com/JohnnyMorganz/luau-lsp/issues/743#issuecomment-2293315812
+  patches = [ ./cmakelists.patch ];
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
```

Comment:
```suggestion
```
That's default, see https://github.com/NixOS/nixpkgs/blob/0f3dfda1e5b65541000b3f03109da3bd3634bc9c/pkgs/by-name/cm/cmake/setup-hook.sh#L109